blockchain/ethereum/contracts/erc1155/erc1155 abi/aa6191bb-c26c-49d0-a597-93bd97294506.json

```diff
@@ -49,7 +49,7 @@
     }
   },
   "iconId": "11e997ab-33d0-473a-aa56-1e2fe4d30d5c",
-  "description": "Holds the ABI of the ERC721 contract.",
+  "description": "Holds the ABI of the ERC1155 contract interface.",
```

Comment:
Is the "interface" term required? It seems like a duplicate to me as the term ABI already contains it.